public/data/aaquibdev.json

```
@@ -1,38 +1,38 @@
 {
-  "name": "Aaquib Ali",
-  "type": "personal",
-  "bio": "Improving the world w/ better education...",
-  "avatar": "https://github.com/aaquibdev.png",
-  "links": [
-    {
-      "name": "Blogs",
-      "url": "https://aaquibdev.medium.com/",
-      "icon": "book"
-    },
-    {
-      "name": "Follow me on Twitter",
-      "url": "https://twitter.com/aaquib_dev",
-      "icon": "twitter"
-    },
-    {
-      "name": "Check out my portfolio",
-      "url": "https://aaquibdev.github.io/Aaquib-Ali-Portfolio/",
-      "icon": "globe"
-    },
-    {
-      "name": "Connect with me on LinkedIn",
-      "url": "https://www.linkedin.com/in/aaquib-ali-39494b210/",
-      "icon": "linkedin"
-    },
-    {
-      "name": "Check out my GitHub",
-      "url": "https://github.com/aaquibdev",
-      "icon": "github"
-    },
-    {
-      "name": "Follow me on Instagram",
-      "url": "https://www.instagram.com/aaquib.dev/",
-      "icon": "instagram"
-    }
-  ]
-}
+    "name": "Aaquib Ali",
+    "type": "personal",
+    "bio": "Improving the world w/ better education...",
+    "avatar": "https://github.com/devaaquib.png",
+    "links": [
+      {
+        "name": "Blogs",
+        "url": "https://aaquibdev.medium.com/",
+        "icon": "book"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/dev_aaquib",
+        "icon": "twitter"
+      },
+      {
+        "name": "Check out my portfolio",
+        "url": "https://devaaquib.github.io/Aaquib-Ali-Portfolio/",
+        "icon": "globe"
+      },
+      {
+        "name": "Connect with me on LinkedIn",
+        "url": "https://www.linkedin.com/in/aaquib-ali-39494b210/",
+        "icon": "linkedin"
+      },
+      {
+        "name": "Check out my GitHub",
+        "url": "https://github.com/devaaquib",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Instagram",
+        "url": "https://www.instagram.com/aaquib.dev/",
+        "icon": "instagram"
+      }
+    ]
+  }
```

Missing an endline. It's best to keep it as it could cause issues with some IDEs and configurations.
```suggestion
  }

```